include/zephyr/net/socket_service.h

```diff
@@ -32,15 +32,23 @@
 extern "C" {
```

Comment:
Also please change `net: sockets: socket_service: remove k_work related code` to
`net: sockets: service: Remove k_work related code`